local-build-plugins/src/main/java/org/hibernate/orm/jakarta/JakartaDirectoryTransformation.java

```diff
@@ -1,82 +0,0 @@
-/*
- * SPDX-License-Identifier: Apache-2.0
- * Copyright Red Hat Inc. and Hibernate Authors
- */
-package org.hibernate.orm.jakarta;
-
-import java.io.File;
-import javax.inject.Inject;
-
-import org.gradle.api.DefaultTask;
-import org.gradle.api.file.DirectoryProperty;
-import org.gradle.api.model.ObjectFactory;
-import org.gradle.api.tasks.CacheableTask;
-import org.gradle.api.tasks.InputDirectory;
-import org.gradle.api.tasks.OutputDirectory;
-import org.gradle.api.tasks.PathSensitive;
-import org.gradle.api.tasks.PathSensitivity;
-import org.gradle.api.tasks.TaskAction;
-
-import static org.hibernate.orm.jakarta.JakartaPlugin.JAKARTA;
```

Comment:
That's a nice way to resolve any incompatibilities with Gradle 9 in this task 🥳 🙂 !